src/scs/scs.md

```diff
@@ -1,3 +1,7 @@
 # SCS
 
+Supply Chain Security is a process of securitng the machinery of the development, build and release environment, in other words, securing every component that a software artifact might be touching on its way from the developer towards the consumer. The software artifact should be encrypted on each possilbe transition phase and the integrity of it should be verified at each destination.
```

Comment:
The first sentence is too long. Let's do the following:

1. Let's end the first sentence with the words "release environment". Also, please change it to the following: Supply Chain Security (SCS) is a process of **securing** the machinery of the development, **building** and release environment.

2. The next sentence should be something like this: That means every component that a software artifact might be touching on its way from the developer to the consumer will be secured.